hedera-mirror-rest/__tests__/specs/contracts-result-details-01-timestamp.spec.json

```
@@ -17,6 +17,27 @@
         "payer_account_id": 8001
       }
     ],
+    "contractlogs": [
+      {
+        "consensus_timestamp": 167654000123456,
```

```suggestion
        "consensus_timestamp": "167654000123456",
```
consensus_timestamp can be larger than js Number MAX_SAFE_INTEGER, so we usually use a string everywhere in the json spec files.